backend-processing.ts

```diff
@@ -96,24 +99,65 @@ export default class BackendProcessing {
     }
     let url = entry["hyperlink url"];
     let comments = entry["comments"];
-    database.insertFromCsvLine(
-      userID,
-      fundID,
-      firstCompany,
-      name,
-      firstPosition,
-      sterm,
-      eterm,
-      employer,
-      position,
-      url,
-      comments
-    );
+    database
+      .insertFromCsvLine(userID, fundID, firstCompany, name, firstPosition, sterm, eterm, employer, position, url, comments)
+      .then((individualID) => {
+        let q1 = entry["q1 " + year];
+        let q2 = entry["q2 " + year];
+        let q3 = entry["q3 " + year];
+        let q4 = entry["q4 " + year];
+        this.processQuarter(q1, 1, individualID, userID, fundID, year);
+        this.processQuarter(q2, 2, individualID, userID, fundID, year);
+        this.processQuarter(q3, 3, individualID, userID, fundID, year);
+        this.processQuarter(q4, 4, individualID, userID, fundID, year);
+      });
+  }
+  processQuarter(quarterString, quarterNumber, individualID, userID, fundID, year) {
+    if (quarterString == null || quarterString == "") return;
```

Comment:
I think this is how we ignore blank sections of quarter slots in the csv. Good job.